src/store/sessions-slice.js

```diff
@@ -0,0 +1,24 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+let accountsInLocalStorage = JSON.parse(localStorage.getItem("sessions"));
+let isAnAccountLoggedIn = JSON.parse(localStorage.getItem("isAnAccountLogged"))
+
+export const sessionsSlice = createSlice({
+    name: "sessions",
+    initialState: {
+        accountsLogged: accountsInLocalStorage ? accountsInLocalStorage : [],
```

Comment:
what do you keep in `accountsLogged`? all the registered user accounts? if so let's rename it to `users`